modules/lvms-about-scaling-storage-of-clusters.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-about-scaling-storage-of-cluster_{context}"]
+= About scaling up the storage of clusters
+
+{product-title} supports additional worker nodes for clusters on bare-metal user-provisioned infrastructure. {lvms} detects and uses additional worker nodes when the nodes become active.
+
+To scale up the storage capacity of the worker nodes on a cluster, you must add the path to the new device in the `deviceSelector` field of the `LVMCluster` custom resource (CR).
```

Comment:
There is also the option of not specifying any `deviceSelector` in which case LVMS picks up any new device automatically if they are supported. I think we should also mention it here.

Reply:
Added this info.